app/controllers/application_controller.rb

```diff
@@ -1,5 +1,2 @@
 class ApplicationController < ActionController::Base
-  def current_user
-    @current_user = User.first
-  end
```

Comment:
## [OPTIONAL]

- [ ] Nice job on authenticating the user however I can see all your routes are available for an unauthorized user. Kindly I would suggest you to block access to unauthorized users and to do that you just need to require the user to be authenticated before any action in your controllers. Hint :point_down: 


```rb
class ApplicationController < ActionController::Base
  before_action :authenticate_user!
end
```